examples/pytorch/question-answering/run_seq2seq_qa.py

```diff
@@ -476,15 +483,18 @@ def preprocess_validation_function(examples):
             max_length=max_seq_length,
             padding=padding,
             truncation=True,
-            return_overflowing_tokens=True,
             return_offsets_mapping=True,
+            # return_overflowing_tokens=True,
         )
-        # Tokenize targets with the `text_target` keyword argument
-        labels = tokenizer(text_target=targets, max_length=max_answer_length, padding=padding, truncation=True)
+
+        # Setup the tokenizer for targets
+        with tokenizer.as_target_tokenizer():
+            labels = tokenizer(targets, max_length=max_answer_length, padding=padding, truncation=True)
```

Comment:
No, this API is deprecated. Please revert this change.

Reply:
I see